core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -696,5 +696,24 @@ class MapUnit {
         }
     }
 
+    fun matchesCategory(category:String): Boolean {
```

Comment:
DO NOT allow lowercasing - the translation is case-sensitive, and so the uniques should be too